examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,54 @@
+"""
+===========================
+WCS Axes Plotting
+===========================
+In this example we are going to look at how pixel and map coodinates work with WCS Axes.
+"""
+
+import matplotlib.pyplot as plt
+
+from astropy import units as u
+
+import sunpy.data.sample
+import sunpy.map
+
+###############################################################################
+# Lets start by creating a Map from the sample data.
+my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+###############################################################################
+# Lets choose xlimits and ylimits in arcsecs that we are interested in.
+xlims_world = [500, 1100]*u.arcsec
+ylims_world = [-800, 0]*u.arcsec
+
+###############################################################################
+# By setting the projection of the axis, a WCSAxes is created and this enables
+# the plot to be created with the map coordinates, in this case helioprojective
+# coordinates. The standard plot
+# command expects pixel coordinates. It is also possible to plot a point using the
+# map coordinates if we pass the transformation but this transform expects
+# the values to be in degrees and not arcseconds.
+fig = plt.figure()
+ax = fig.add_subplot(projection=my_map)
+my_map.plot(axes=ax, clip_interval=(1, 99.9)*u.percent)
+
+# plot a point in the middle of the image
+# we are plotting the pixel coordinates
+pixel_coord = [my_map.data.shape[0]/2., my_map.data.shape[1]/2.] * u.pix
+ax.plot(pixel_coord[0], pixel_coord[1], 'x', color='white',
+        label=f'Pixel coordinate [{pixel_coord[0]}, {pixel_coord[1]}]')
+ax.coords.grid(color='yellow', linestyle='solid', alpha=0.5)
+
+map_coord = ([-300, 200] * u.arcsec)
+# Using the transform command expects coordinates in degrees and not arcseconds
+ax.plot(map_coord[0].to('deg'), map_coord[1].to('deg'), 'o', color='Blue',
+        transform=ax.get_transform('world'),
+        label=f'Map coordinate [{map_coord[0]}, {map_coord[1]}]')
+
+# We can now plot a map showing WCSaxes coordinates
+ax.scatter(500*u.arcsec.to(u.deg), 100*u.arcsec.to(u.deg), color='Red',
+           transform=ax.get_transform("world"),
+           label=f'WCS coordinate [{map_coord[0]}, {map_coord[1]}]')
```

Comment:
```suggestion
           label=f'WCS coordinate [{500*u.arcsec}, {100*u.arcsec}]')
```

Reply:
Oh! Silly me!! it's WCSAxes, arcsec should be used. Thanks a bunch Albert!